### PR TITLE
docs: add link of universe docs

### PR DESCRIPTION
**Pull request by shmpwk**
Signed-off-by: Shumpei Wakabayashi <shumpei.wakabayashi@tier4.jp>

## Description

IMO, universe docs https://autowarefoundation.github.io/autoware.universe/main/ is not well known 
because it is only linked at https://autowarefoundation.github.io/autoware-documentation/main/support/docs-guide/  and few people may find it.

In order to encourage people to find it,
I propose

1. Paste universe docs link here 

無題のプレゼンテーション

2.  Write link as this PR.

3.  Write link in https://autowarefoundation.github.io/autoware-documentation/main/design/ by https://github.com/autowarefoundation/autoware-documentation/pull/86#discussion_r870009574

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

 [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by kenji-miyake** [comment]
I added the URL to the About section.
image

But it's also good to add the link in README. :+1:

File: README.md [hunk 1]
```diff
@@ -4,4 +4,6 @@ This is one of the prototype repositories for Autoware Core/Universe that AWF ag
 
 Please see [autowarefoundation/autoware](https://github.com/autowarefoundation/autoware) for more details.
 
+Also see [Autoware Universe Documentation](https://github.com/autowarefoundation/autoware.universe).
```
Comment (kenji-miyake) [inline README.md:7]: How about this? BTW, you can delete the existing sentences.

```md
For Autoware's general documentation, see [Autoware Documentation](https://autowarefoundation.github.io/autoware.universe/).

For detailed documents of Autoware Universe components, see [Autoware Universe Documentation](https://autowarefoundation.github.io/autoware.universe/).
```

Comment (shmpwk) [reply README.md]: Oh i mistakenly use wrong link, thank you.

**Comment by kenji-miyake** [comment]
FYI, a reference about link text. https://developers.google.com/style/link-text

File: README.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 This is one of the prototype repositories for Autoware Core/Universe that AWF agreed to create in the [TSC meeting on 2021/11/17](https://discourse.ros.org/t/technical-steering-committee-tsc-meeting-36-2021-11-17-minutes/23168).
```
Comment (kenji-miyake) [inline README.md:3]: Could you remove this as well? I think we don't need it anymore.

```suggestion
```

**Comment by kenji-miyake** [comment]
@LalithVipulananthan Could you check this? :pray:

**Review by kenji-miyake — APPROVED** [review]

**Comment by LalithVipulananthan** [comment]
In the main Autoware documentation, perhaps we need to explain the difference between the main Autoware documentation and the Universe documentation?

**Comment by shmpwk** [comment]
@LalithVipulananthan 
You mean this PR cannot explain the difference of the Autoware documentaion and Autoware universe documentation?

**Review by LalithVipulananthan — APPROVED** [review]
This looks fine for now. We should also make sure that we update the README files for other repositories in the AWF org too.

**Comment by shmpwk** [comment]
@LalithVipulananthan 
> We should also make sure that we update the README files for other repositories in the AWF org too.
I agree and I will make another PR.

**Comment by LalithVipulananthan** [comment]
@shmpwk I had a check and there's only a few repositories that need to be updated:
- https://github.com/autowarefoundation/autoware_common (already submitted a PR to update this one)
- https://github.com/autowarefoundation/autoware.core
- https://github.com/autowarefoundation/autoware_launch
- https://github.com/autowarefoundation/autoware_common

**Comment by shmpwk** [comment]
> I had a check and there's only a few repositories that need to be updated:

Thank you. I made thier PR.
https://github.com/autowarefoundation/autoware_launch/pull/47
https://github.com/autowarefoundation/autoware.core/pull/36
https://github.com/autowarefoundation/autoware_common/pull/78

**Pull request merged**